reframe/frontend/cli.py

```diff
@@ -53,6 +53,13 @@ def format_check(check, detailed):
     return '\n'.join(lines)
 
 
+def format_env(envvars):
+    ret = 'ReFrame environment variables:\n'
```

Comment:
```suggestion
    ret = '[ReFrame Environment]\n'
```

So as to match the `[ReFrame setup]`.